src/main/resources/db/migration/V20220214_51__HMAN-51-PartiesNotified.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE ONLY public.hearing_response
+    ADD COLUMN IF NOT EXISTS response_version varchar(60) not null;
+
+ALTER TABLE ONLY public.hearing_response
```

Comment:
hearing_request was no longer required for my ticket, not sure if its still relevant for yours

Reply:
did you mean request_version?